WebApiBackend/WebApiBackend/Controllers/FlatController.cs

```diff
@@ -132,5 +131,50 @@ public ActionResult<FlatDTO> createFlat()
             Response.StatusCode = 201;
             return new FlatDTO(flat);
         }
+
+        /// <summary>
+        /// Delete Method -Remove a member out of a flat
+        /// </summary>
+        /// <param name="deleteUserName">The username of a member being removed from the flat</param>
+        /// <response code="202">The user have successfully remove her/himself from the flat, and be redirected to create flat page</response>
+        /// <returns>The response body is the flat members </returns>
+
+        [Authorize]
+        [HttpDelete("{deleteUserName}")]
```

Comment:
Probably want to add API docs for these new endpoints. Would appreciate someone from the Flat Members team having a look and proper test, I just got the PR ready to be merged in.